src/repo.rs

```diff
@@ -1020,6 +1020,24 @@ impl Repository {
             Ok(Oid::from_raw(&raw))
         }
     }
+
+    /// Calculate how far ahead and/or behind a `local` commit is compared to `upstream`
```

Comment:
I typically just copy/paste the documentation in libgit2 itself, could this also include the extended docs?

https://libgit2.github.com/libgit2/#HEAD/group/graph/git_graph_ahead_behind